src/sketchxport-export.js

```diff
@@ -0,0 +1,44 @@
+// documentation: https://developer.sketchapp.com/reference/api/
+import { startOfPlugin, endOfPlugin } from './utils/debug';
+import playSystemSound from './utils/playSystemSound';
+import sketchConfig, { sketchAlert } from './utils/sketchConfig';
+
+const debugConfig = {
+  debugEnabled: true,
+  withSuccessSound: true,
+  withFailureSound: true,
+};
+
+export default function(context) {
+  startOfPlugin(true);
+
+  const { doc } = sketchConfig(context);
```

Comment:
what do you do in case of no sketchConfig? shouldn't you call the sketchxport-configure script?